run_python_script.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -l
+
+## Nazwa zlecenia
+#SBATCH -J Some_name
+
+## Liczba alokowanych węzłów
+#SBATCH -N 1
+
+## Liczba zadań per węzeł (domyślnie jest to liczba alokowanych rdzeni na węźle)
+#SBATCH --ntasks-per-node=1
+
+## Ilość pamięci przypadającej na jeden rdzeń obliczeniowy (domyślnie 5GB na rdzeń)
+#SBATCH --mem-per-cpu=15GB
+
+## Maksymalny czas trwania zlecenia (format HH:MM:SS)
+#SBATCH --time=01:00:00
+
+## Nazwa grantu do rozliczenia zużycia zasobów
+#SBATCH -A plgpreludium
+
+## Specyfikacja partycji
+#SBATCH -p plgrid-gpu-v100
+#SBATCH --gres=gpu
+
+## Plik ze standardowym wyjściem
+#SBATCH --output="output_0.out"
+
+## Plik ze standardowym wyjściem błędów
+#SBATCH --error="error_0.err"
+
+
+## przejscie do katalogu z ktorego wywolany zostal sbatch
+cd $SLURM_SUBMIT_DIR
+
+srun /bin/hostname
+
+export LD_LIBRARY_PATH=/net/people/plgztabor/cuda/lib64:$LD_LIBRARY_PATH
+module load plgrid/libs/tensorflow-gpu/2.2.0-python-3.7
+pip3 install -r bcnn/requirements.txt
+python3 $1
```

Comment:
```suggestion

module load plgrid/tools/python/3.8
module load plgrid/apps/cuda/11.2

pip3 install -r bcnn/requirements.txt

python3 $1
```

Reply:
To powinno pomóc przy tym błędzie z TFP, ale nie jestem teraz tego w stanie sprawdzić, bo dostaję "sbatch: error: Batch job submission failed: Invalid account or account/partition combination specified". :/

Reply:
Pewnie dlatego, że właśnie wczoraj skończył nam się grant.
![image](https://user-images.githubusercontent.com/21079319/134584285-233bd19d-cff0-44ac-8b7f-53406ba645ef.png)